### DISCUSSION TITLE
Question about collections

### DISCUSSION THREAD

**Discussion by froyglez** [metabrainz/musicbrainz-android#223] [Q&A] [answered]
What are collections used for? I want to use a tool that helps me record the songs, albums, and artists I like. Are the collections suitable for that? Or are they intended for another use?

**Comment by 07jasjeet** [comment, marked as answer]
Hello there! By "record the songs", do you mean when songs are played on your phone, the app should automatically record those songs? A music scrobbler? 
If yes, then please check out ListenBrainz Android (available on play store) for that purpose!

**Reply by froyglez** [reply]
Hello, I don't want to record my listening history, rather I want to record the songs, albums and artists I like :)

**Reply by froyglez** [reply]
Sorry for my bad english. Maybe some better words would be "track", register" or "list" instead of "record"

**Reply by froyglez** [reply]
I think my question was solved in the MetaBrainz Community discourse. Thank you so much!

**Reply by froyglez** [reply]
And yes, the ListenBrainz app seems interesting. I'll take a look of it! Than you!

**Reply by 07jasjeet** [reply]
Good to hear!!
